Command/AbstractGenerateCommand.php

```diff
@@ -59,4 +60,17 @@ protected function getTotalCount(array $config)
 
         return $totalCount;
     }
+
+    /**
+     * Create the output dir where fixtures will be generated if necessary.
+     *
+     * @param string $outputDir
+     */
+    protected function createOutputDirIfNecessary($outputDir)
+    {
+        $fs = new Filesystem();
+        if (!$fs->exists($outputDir)) {
+            $fs->mkdir($outputDir);
```

Comment:
Hum, bad practice !
Your software shouldn't do more than it's expected to do.

And creating directory is not its responsability. Look at Unix command: apart mkdir or command with directory modificator (cp for exemple), no command will silently create a directory without user knowledge: they will fail if the directory does not exists. Also, it's the best way for the user to do not understand why the directory he has created is not populated as he made a typo in the config file and the command didn't tell him !